modules/apps/27-interchain-accounts/controller/keeper/account.go

```diff
@@ -51,12 +52,14 @@ func (k Keeper) RegisterInterchainAccount(ctx sdk.Context, connectionID, owner s
 		icatypes.TxTypeSDKMultiMsg,
 	)
 
-	versionBytes, err := icatypes.ModuleCdc.MarshalJSON(&metadata)
+	versionbytes, err := icatypes.ModuleCdc.MarshalJSON(&metadata)
 	if err != nil {
 		return err
 	}
+	version := string(versionbytes)
+
+	msg := channeltypes.NewMsgChannelOpenInit(portID, version, channeltypes.ORDERED, []string{connectionID}, icatypes.PortID, authtypes.NewModuleAddress(icatypes.ModuleName).String())
 
```

Comment:
Changes in this file should be discarded

Reply:
ok